packages/scaffold/src/views/w3m-transactions-view/index.ts

```diff
@@ -52,8 +51,10 @@ export class W3mTransactionsView extends LitElement {
             }
           }
         }),
+        NetworkController.subscribe(() => {
```

Comment:
This should probably be a subscribeKey to a specific field, I think caipChain, so it re-fetches only when that one change happens vs if anything else in controller changes.